src/deploy/functions/params.ts

```diff
@@ -66,29 +54,22 @@ export function resolveString(
   from: string | build.Expression<string>,
```

Comment:
looks like we should update the docstring for this fn. Likewise for other `resolve` functions

Reply:
Done.